app/components/candidate_interface/course_choices_review_component.rb

```diff
@@ -55,30 +53,6 @@ def any_withdrawable?
       end
     end
 
-    def show_missing_banner?
-      @show_incomplete && !@application_form.course_choices_completed && @editable
-    end
-
-    def course_change_path(application_choice)
-      if multiple_courses?(application_choice) || application_choice.course_full?
```

Comment:
Shouldn't the `multiple_courses?` and `multiple_sites?` methods be deleted as well?